packages/deployer/test/fixtures/completed-deployment/deployments/local/official/2021-12-23-00.json

```diff
@@ -0,0 +1,127 @@
+{
```

Comment:
Should these files be ignored?

Reply:
Nope, this are just new fixtures with the new format for the tests